assets/setup-wizard/features/confirmation-modal.js

```diff
@@ -43,19 +40,6 @@ const ConfirmationModal = ( {
 		) }
 		isDismissible={ false }
 	>
-		<List
```

Comment:
For now, I just removed this part to remove the component that we won't use anymore based on the new design/behavior. But the whole Features step will be removed later or changed to some of the other steps.